### PR TITLE
[terra-docs] Fix placeholder styling

### LINKED ISSUE #3450: [terra-docs] Themes failing due to color-contrast issue due to placeholder styling.
# Bug Report
## Description

While upgrading themes to the latest snapshots I am seeing color-contrast issue due to placeholder styling

## Expected Behavior

Themes shouldn't fail.

## Possible Solution

Upgrade placeholder styling

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5]
* Webpack Version:
* Operating System and version (desktop or mobile):

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

This PR adds new theme variables and updates styles to fix placeholder color-contrast issues in themes.

Closes #3450 

### Deployment Link

https://terra-core-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-docs/src/Placeholder.module.scss [hunk 1]
```diff
@@ -22,5 +22,13 @@
     color: var(--terra-docs-placeholder-inner-color, #787575);
     display: flex;
     justify-content: center;
+
+    p {
```
Comment (mjhenkes) [inline packages/terra-docs/src/Placeholder.module.scss:26]: use a class instead of styling the p tag

Comment (pranav300) [reply packages/terra-docs/src/Placeholder.module.scss]: Updated here https://github.com/cerner/terra-core/pull/3452/commits/05fb3a469732f8968f1c93a9459db6d2d7ad7b62.

File: packages/terra-docs/src/Placeholder.module.scss [hunk 2]
```diff
@@ -22,5 +22,13 @@
     color: var(--terra-docs-placeholder-inner-color, #787575);
     display: flex;
     justify-content: center;
+
+    .title {
+      background-clip: padding-box;
+      background-color: var(--terra-docs-placeholder-inner-p-background-color, #fff);
```
Comment (mjhenkes) [inline packages/terra-docs/src/Placeholder.module.scss:28]:
```suggestion
      background-color: var(--terra-docs-placeholder-inner-title-background-color, #fff);
```

Comment (pranav300) [reply packages/terra-docs/src/Placeholder.module.scss]: Updated here https://github.com/cerner/terra-core/pull/3452/commits/2912579979a51b5828230bb8d337135f781debcc.

**Review by benbcai — APPROVED** [review]

**Review by ShettyAkarsh — APPROVED** [review]

**Review by dkasper-was-taken — APPROVED** [review]

**Review by supreethmr — APPROVED** [review]

**Review by lokesh-0813 — APPROVED** [review]

**Pull request merged**